src/NServiceBus.AcceptanceTests/Core/Installers/When_installers_not_enabled.cs

```diff
@@ -28,7 +29,8 @@ class EndpointWithInstaller : EndpointConfigurationBuilder
         {
             public EndpointWithInstaller()
             {
-                EndpointSetup<DefaultServer>(c => c.DisableInstallers());
+                // disable installers as installers are enabled by default in DefaultServer
+                EndpointSetup<DefaultServer>(c => c.GetSettings().Set("Installers.Enable", false));
```

Comment:
I guess this is OK, but I'd almost prefer to see it use another template so that we didn't have to do something special here.